nodes/widgets/ui_chart.html

```diff
@@ -80,11 +80,41 @@
                 xAxisType: { value: 'time' },
                 xAxisFormat: { value: '' },
                 xAxisFormatType: { value: 'auto' },
+                xmin: {
+                    value: '',
+                    validate: function (value) {
+                        if (this.chartType === 'histogram' && this.xAxisType === 'bins') {
+                            return typeof value !== 'undefined' && value !== '' && RED.validators.number()
```

Comment:
this does not validate the field correctly. 

e.g. I can enter  a bunch of spaces `  ` or even  `  aaa  ` and the field is not marked invalid, deploys without warning and the chart fails to show any data (and no errors are generated client side either)

![chrome_RlWHJ5kLlD](https://github.com/user-attachments/assets/60d89fe8-a405-4f11-a677-1154f36da90a)

Reply:
I have prepared a change locally and will push shortly to address the ineffective validations

In short, the RED.validator.number incorrectly returns `true` for certain scenarios.


### fixed locally demo
![chrome_pCZbYR2TPP](https://github.com/user-attachments/assets/eca2170a-e5d9-4eb8-ade8-acaa7905e3b0)

Reply:
fixed number validation in [fix number validation](https://github.com/FlowFuse/node-red-dashboard/pull/1357/commits/39ec9544a8d176885471ec61cf1d420331e756b1)  [39ec954](https://github.com/FlowFuse/node-red-dashboard/pull/1357/commits/39ec9544a8d176885471ec61cf1d420331e756b1)